src/sidebar/store/modules/groups.js

```diff
@@ -156,44 +180,22 @@ const getMyGroups = createSelector(
   }
 );
 
-/**
- * Return groups the user isn't a member of that are scoped to the URI.
- *
- * @return {Group[]}
- */
-const getFeaturedGroups = createSelector(
-  state => state.groups.groups,
-  groups => groups.filter(group => !group.isMember && group.isScopedToUri)
-);
-
 /**
  * Return groups that don't show up in Featured and My Groups.
  *
  * @return {Group[]}
  */
 const getCurrentlyViewingGroups = createSelector(
```

Comment:
I wonder if we should comment root selectors at their point of definition. It would be hard to scan this module and immediately see that `state` here is `rootState` instead of local. At the very least, perhaps the `createSelector` input selectors could take `rootState` as their argument...?

Reply:
I did re-organize these selectors so that the root selectors are at the bottom of the file. Changing `state` => `rootState` seems like a good idea though.